internal/js/modules/k6/grpc/client_test.go

```diff
@@ -1597,3 +1606,16 @@ func TestClientConnectionReflectMetadata(t *testing.T) {
 
 	assert.True(t, foundReflectionCall, "expected to find a reflection call in the logs, but didn't")
 }
+
+func extractTypesFromClientAsSobekValue(t *testing.T, val sobek.Value) *protoregistry.Types {
+	t.Helper()
+
+	client, ok := val.Export().(*k6grpc.Client)
+	require.True(t, ok, "got: %T, want *k6grpc.Client", client)
+
+	clientValue := reflect.ValueOf(client).Elem()
+	typesField := clientValue.FieldByName("types")
+
+	typesField = reflect.NewAt(typesField.Type(), unsafe.Pointer(typesField.UnsafeAddr())).Elem()
+	return typesField.Interface().(*protoregistry.Types)
```

Comment:
I know this type of magic is generally discouraged, but I hope it's fine as long as it's used only for testing purposes.

The other alternatives I could think of were:
- Renaming the test package name, from `grpc_test` to `grpc` to have access to `Client` private attributes/methods, _but I think it's useful to leave it as-is as long as we can._ 
- Exposing the `types` attribute or some of its methods publicly, _which at the beginning I thought it could also be a valid approach as long as the package lives under `internal`, but then realized that public methods are automatically exposed to the JS layer, so likely a no-go._ 

So, finally I decided that this is the least painful approach.